lightning/src/ln/channelmanager.rs

```diff
@@ -8385,8 +8385,10 @@ macro_rules! create_refund_builder { ($self: ident, $builder: ty) => {
 
 /// Defines the maximum number of [`OffersMessage`] to be sent along different reply paths.
 /// Sending multiple invoices increases the chances of successful delivery in case some
-/// paths are unavailable. However, only one invoice for a given payment ID will be paid,
+/// paths are unavailable. However, only one invoice for a given [`PaymentId`] will be paid,
 /// even if multiple invoices are received.
+/// 
+/// [`PaymentId`]: crate::ln::channelmanager::PaymentId
```

Comment:
This shouldn't be needed since the type is in this file.

Reply:
Right! Makes sense!
Fixed in [0354d8d](https://github.com/lightningdevkit/rust-lightning/commit/0354d8dd8ad2608400be58c4de6e5d6c037df5af)
Thanks!